en.lproj/Localizable.strings

```diff
@@ -387,6 +387,4 @@ Widgets
 Units: ºC, ºF, metric, miles
 
 • SUPPORT & MORE
-If you have any remarks and just to say hi, feel free to contact us at support@shoobie.app or on Twitter at @shoobieapp";
-
-
+If you have any remarks or just to say hi, feel free to contact us at support@shoobie.app or on Twitter at @shoobieapp";
```

Comment:
If you have any remarks or just want to say hi, feel free to contact us at support@shoobie.app or on Twitter at @shoobieapp